content/en/home/timeline.md

```diff
@@ -18,9 +18,27 @@ date_format = "January 2006"
 #   Required fields are `title`, `company`, and `date_start`.
 #   Leave `date_end` empty if it's your current employer.
 #   Begin/end multi-line descriptions with 3 quotes `"""`.
+[[experience]]
+  title = "v1.0.0"
+  company = "KubeEdge v1.0.0 release"
+  company_url = "https://github.com/kubeedge/kubeedge/releases/tag/v1.0.0"
+  location = "June, 2019"
+  date_start = ""
+  date_end = ""
+  description = """
+**Features added**
+
+- Edge mesh provides service mesh capabilities on the edge. Pod-to-pod communication on the same edge node or across edge nodes in the same subnet is supported. <a href='https://github.com/kubeedge/kubeedge/blob/master/docs/proposals/edgemesh-design.md' target="_blank">&hellip; Read more</a>
+- CRI-compliant runtimes like containerd are supported at the edge. This enables edged to communicate with the runtime to manage containers running on resource constrained edge nodes.
+- Enhanced cloud and edge communication efficiency via QUIC, a UDP-based protocol. CloudHub supports both Websocket and QUIC protocol access at the same time. The edgehub can choose one of the protocols to access the cloudhub. <a href='https://github.com/kubeedge/kubeedge/blob/master/docs/proposals/quic-design.md' target="_blank">&hellip; Read more</a>
+- Modbus Mapper to connect and control devices that use Modbus(RTU/TCP) as a communication protocol.<a href='https://github.com/kubeedge/kubeedge/blob/master/docs/mappers/modbus_mapper.md'target="_blank">&hellip; Read more</a>
+- Edge Site enables to run a standalone Kubernetes cluster at the edge along with KubeEdge to get full control and improve the offline scheduling capability. <a href='https://github.com/kubeedge/kubeedge/blob/master/docs/proposals/EdgeSite.md' target="_blank">&hellip; Read more</a>
```

Comment:
Same as above. Please update the links to the appropriate urls from the docs website.
Proposals is internal and not available in a readable format. If this needs to be shown to the users then move the proposals file to the docs folder.

Reply:
done.